src/openrct2-ui/windows/Options.cpp

```diff
@@ -221,6 +222,7 @@ static rct_widget window_options_display_widgets[] = {
     MakeWidget        ({155, 176}, {136,  12}, WWT_CHECKBOX, WindowColour::Secondary, STR_MULTITHREADING,                    STR_MULTITHREADING_TIP                   ), // Multithreading
     MakeWidget        ({ 11, 176}, {143,  12}, WWT_CHECKBOX, WindowColour::Secondary, STR_USE_VSYNC,                         STR_USE_VSYNC_TIP                        ), // Use vsync
     MakeWidget        ({ 11, 191}, {280,  12}, WWT_CHECKBOX, WindowColour::Secondary, STR_MINIMISE_FULLSCREEN_ON_FOCUS_LOSS, STR_MINIMISE_FULLSCREEN_ON_FOCUS_LOSS_TIP), // Minimise fullscreen focus loss
+    MakeWidget        ({ 11, 206}, {280,  12}, WWT_CHECKBOX, WindowColour::Secondary, STR_ENABLE_SCREENSAVER,                STR_ENABLE_SCREENSAVER                   ), // Enable screensaver
```

Comment:
Should we have a tooltip with a bit more information?

Reply:
Also tooltips (for now) need to specify that their size and colour so should always be a separate string.

Reply:
Corrected. Please see second image attached.